tlslite/tlsrecordlayer.py

```diff
@@ -147,6 +147,9 @@ def __init__(self, sock):
         #Limit the size of outgoing records to following size
         self.recordSize = 16384 # 2**14
 
+        # Indicator for heartbeat extension
+        self.heartbeat = False
```

Comment:
oh, and probably `heartbeatSupported` is also needed (as, IIRC, receiving heartbeat if the extension was negotiated but peers are not allowed to send requests should not cause the connection to be dropped)